Extensions/xkit_preferences.js

```diff
@@ -1092,9 +1125,16 @@ XKit.extensions.xkit_preferences = new Object({
 			m_html = m_html + '<div class="more-info" data-more-info="' + obj.details + '">more info</div>';
 		}
 
+
+		var install_button_text = "Install";
+		if(XKit.installed.check(obj.name)) { install_button_text = "Installed"; }
+
+		var install_button_installed_class = "";
+		if(XKit.installed.check(obj.name)) { install_button_installed_class = " installed"; }
```

Comment:
don't need two classes, and this leads to double spaces. should just stick to `xkit-installed-extension`